scripts/akrtma3d/exakrtma3d_gsianl.ksh

```diff
@@ -385,8 +408,8 @@ done
 #${MV} ${SFCOBS_USELIST}/current_mesonet_uselist.txt ${SFCOBS_USELIST}/${PDYm1}_mesonet_uselist.txt
 #scpreq Edward.Colon@dtn-jet.boulder.rdhpcs.noaa.gov:/mnt/lfs4/HFIP/hfv3gfs/Edward.Colon/reject_use_lists/current_mesonet_uselist.txt ${SFCOBS_USELIST}/
 #fi
-${CP} ${AIRCRAFT_REJECT}/current_bad_aircraft.txt current_bad_aircraft
-${CP} ${SFCOBS_USELIST}/current_mesonet_uselist.txt gsd_sfcobs_uselist.txt
+#${CP} ${AIRCRAFT_REJECT}/current_bad_aircraft.txt current_bad_aircraft
+#${CP} ${SFCOBS_USELIST}/current_mesonet_uselist.txt gsd_sfcobs_uselist.txt
```

Comment:
@MatthewMorris-NOAA @EdwardColon-NOAA @AnnetteGibbs-NOAA 
Hi all,
I understand that Matthew needs to turn off the usage of GSD use/reject list, so he could collect data for AutoQC and run autoqc. But before that, users might still have to use GSD list. So I am thinking about use a option to switch on/off the usage of GSd rej/use list. 
But for this PR, I am OK with turning off GSD list here. Ed, Anntte or I can add the control in the next update.

Thank you!

Reply:
@MatthewMorris-NOAA 
this is just a suggest. we can do it later. So it is good to go here.

Reply:
Hi @GangZhao-NOAA, Currently, I have hard-wired the "i_gsdsfc_uselist" parameter in the parm file. Perhaps, in a future update, we could define that value in the ex-script. Then, an if/else block could determine which lists to copy.

Reply:
OK. I see.